calibre-plugin/plugin-defaults.ini

```diff
@@ -3842,6 +3842,15 @@ fix_fimf_blockquotes:true
 ## chapter text.
 #include_author_notes:false
 
+
+## This option aims to allow easy fetching of the stories on the
+## bookshelf and exclude stories that are not in there.
+## There are 3 options:
```

Comment:
False, true and off feel like a weird set of trinary options.  I know I use the word 'off' as equivalent to 'false' when talking about settings.

Maybe true, false and legacy?